pkg/backupdriver/backup_driver_controller_base.go

```diff
@@ -798,6 +798,19 @@ func (ctrl *backupDriverController) syncDeleteSnapshotByKey(key string) error {
 		return err
 	}
 
+	if delSnapshot.Status.Phase == "" {
+		ctrl.logger.Infof("No Phase set for the DeleteSnapshot CR: %s/%s, will be explicitly update to New", namespace, name)
+		var statusUpdateErr error
+		deleteSnapshotStatusFields := make(map[string]interface{})
+		deleteSnapshotStatusFields["Message"] = "Explicit update to New Phase"
+		delSnapshot, statusUpdateErr = ctrl.updateDeleteSnapshotStatusPhase(context.TODO(), delSnapshot.Namespace, delSnapshot.Name,
+			backupdriverapi.DeleteSnapshotPhaseNew, deleteSnapshotStatusFields)
+		if statusUpdateErr != nil {
+			ctrl.logger.Errorf("Failed to update an unset Phase of DeleteSnapshot %s/%s failed: %v", namespace, name, err)
```

Comment:
Remove "failed" at the end.  It's already mentioned in the beginning.
Mention you try to set to "New"?